guides/V1_Remove_Reaction.md

```diff
@@ -0,0 +1,58 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'remove-a-reaction'
+is: 'future'
```

Comment:
Unclear if, since we're under the `EXPERIMENTAL` header, we should mark this as `experimental` instead of `future`. Same goes for all the others but probably @miguelestrada has a better idea.